scipy/stats/stats.py

```diff
@@ -2980,8 +2980,8 @@ def pearsonr(x, y):
 
     The p-value roughly indicates the probability of an uncorrelated system
```

Comment:
I think one should state the null in a clear way, maybe add: "The null hypothesis of the test is that the correlation coefficient is zero"